src/sidecar/checkArchitecture.ts

```diff
@@ -15,13 +22,7 @@ export function checkSidecarOsAndArch(sidecarPath: string): void {
 
   if (!ourBuild.equals(sidecarBuild)) {
     const errorMsg = `This Confluent extension is built for a different platform (${sidecarBuild.platform}-${sidecarBuild.arch}), whereas your VS Code is on ${ourBuild.platform}-${ourBuild.arch}.`;
-    const button = "Open Marketplace";
-    window.showErrorMessage(errorMsg, button).then((action) => {
```

Comment:
The showErrorMessage with button equivalent is now handled by `showSidecarStartupErrorMessage()`.